### PR TITLE
feat(STONEINTG-558-561) add SLO grafana panels to RHTAP dash

### PR DESCRIPTION
**Pull request by jencull**
Intg Svc SLO 1 panel to RHTAP SLOs
Intg Svc SLO 2a panel to RHTAP SLO's
Intg Svc SLO 5 panel to RHTAP SLO's
Intg Svc SLO 6 panel to RHTAP SLO's

### CODE REVIEW

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 1]
```diff
@@ -58,6 +58,460 @@ data:
...
+                "type": "prometheus",
+                "uid": "${datasource}"
+              },
+              "editorMode": "code",
+              "expr": "sum(rate(integration_svc_response_seconds_bucket{le="30"}[$__rate_interval])) by (job) / sum(rate(integration_svc_response_seconds_count[$__rate_interval]) > 0) by (job)",
+              "legendFormat": "__auto",
+              "range": true,
+              "refId": "A"
+            }
+          ],                                                                                                                #end row one
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:183]: This is not allowed in JSON
```suggestion
          ],
```

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 2]
```diff
@@ -58,6 +58,460 @@ data:
...
+              "refId": "A"
+            }
+          ],                                                                                                               #end row two
+          {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+          },
+          "description": "The objective for the given time window.",
+          "gridPos": {
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:300]: It's suspicious that all graphs have the same grid position

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 3]
```diff
@@ -58,6 +58,460 @@ data:
...
+                    "color": "green",
+                    "value": 95
+                  }
+                ]
+              },
+              "unit": "percentunit"
+            },
+            "overrides": []
+          },
+                    "gridPos": {
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:482]: Indentation

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 4]
```diff
@@ -58,6 +58,460 @@ data:
...
+                "uid": "${datasource}"
+              },
+              "editorMode": "code",
+              "expr": "sum(rate(integration_svc_release_latency_seconds_bucket{le="10"}[$__rate_interval])) by (job) / sum(rate(integration_svc_release_latency_seconds_count[$__rate_interval])) by (job)",
+              "legendFormat": "__auto",
+              "range": true,
+              "refId": "A"
+            }
+          ],                                                                                    #end row 3
+          #end row one
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:404]: this comment is not valid JSON

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 5]
```diff
@@ -58,6 +58,460 @@ data:
...
+                "type": "prometheus",
+                "uid": "${datasource}"
+              },
+              "editorMode": "code",
+              "expr": "sum(rate(integration_svc_release_latency_seconds_bucket{le="10"}[$__rate_interval])) by (job) / sum(rate(integration_svc_release_latency_seconds_count[$__rate_interval])) by (job)",
+              "legendFormat": "__auto",
+              "range": true,
+              "refId": "A"
+            }
+          ],                                                                                    #end row 3
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:403]: Comment on this line is not a valid JSON

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 6]
```diff
@@ -1,1878 +1,2434 @@
-apiVersion: v1
-data:
-  rhtap-slo-dashboard.json: |-
+{
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1]: This is not configmap yaml, you copied here just grafana JSON, not configmap

File: grafana-dashboard-rhtap-slos.configmap.yaml [hunk 7]
```diff
@@ -0,0 +1,2441 @@
+apiVersion: v1
+data:
+  SLOs.json: |-
```
Comment (MartinBasti) [inline grafana-dashboard-rhtap-slos.configmap.yaml:3]: This is configmap yaml, but now this file should be removed from PR

Comment (jencull) [reply grafana-dashboard-rhtap-slos.configmap.yaml]: Ok, I was wondering why there was 2. I don't know how I committed that to root in the first place, rushing on a Friday! Back to one file now. Thanks.

**Comment by jencull** [comment]
image_720_720
image_720

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 8]
```diff
@@ -91,7 +556,7 @@ data:
               "showLineNumbers": false,
               "showMiniMap": false
             },
-            "content": "<h1><center>5xx errors rate for service providers less than 1%</center></h1>",
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:94]: IMHO everything here and below shouldn't be changed, except grid positions

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 9]
```diff
@@ -1866,7 +2371,7 @@ data:
       "timezone": "",
       "title": "RHTAP SLOs",
       "uid": "rhtap-slos",
-      "version": 11,
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1869]: should be version 12

Comment (jencull) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1869]: thanks, done.

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 6 above)
Comment (jencull) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: the config map is at the end of the file

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 8 above)
Comment (jencull) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: agreed. new content there now that reflects this.

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 10]
```diff
@@ -1,8 +1,7 @@
 apiVersion: v1
 data:
   rhtap-slo-dashboard.json: |-
-    {
-      "annotations": {
+    "annotations": {
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:4]: This change looks suspicious, it makes invalid JSON

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 11]
```diff
@@ -1816,7 +2283,7 @@ data:
         "list": [
           {
             "current": {
-              "selected": true,
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1819]: This shouldn't be changed IMHO

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 12]
```diff
@@ -1000,7 +999,6 @@ data:
             },
             {
               "datasource": {
-                "name": "Expression",
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1003]: Is this removal expected?

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 10 above)
Comment (jencull) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:4]: ty.

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 11 above)
Comment (jencull) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: done

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 12 above)
Comment (jencull) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: I went back over all the changes in the commit before last and changed some smaller items that shouldn't have been changed with the new configmap.

**Review by MartinBasti — APPROVED** [review]
Thank you!

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 13]
```diff
@@ -1375,6 +1375,475 @@ data:
...
+              "thresholds": {
+                "mode": "percentage",
+                "steps": [
+                  {
+                    "color": "red",
+                    "value": null
+                  },
+                  {
+                    "color": "green",
+                    "value": 95
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1453]: 90?

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 14]
```diff
@@ -1375,6 +1375,475 @@ data:
...
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1567]: 90?

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 15]
```diff
@@ -1375,6 +1375,475 @@ data:
...
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1681]: 90?

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 16]
```diff
@@ -1375,6 +1375,475 @@ data:
...
```
Comment (MartinBasti) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1795]: 90?

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 13 above)
Comment (jencull) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: done

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 14 above)
Comment (jencull) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: yes

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 15 above)
Comment (jencull) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: yep

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 16 above)
Comment (jencull) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: yes

**Review by MartinBasti — APPROVED** [review]
LGTM

**Comment by jencull** [comment]
@yftacherzog would you mind looking at this? I am finishing my internship on Friday and if I can get this sorted out (even if not merged) before I leave that would be great. Thank you and thank you for all your help and guidance.

**Comment by yftacherzog** [comment]
> @yftacherzog would you mind looking at this? I am finishing my internship on Friday and if I can get this sorted out (even if not merged) before I leave that would be great. Thank you and thank you for all your help and guidance.

@jencull Good luck!

You're probably a bigger expert on those Grafana jsons than I am by now. I tagged Alex and Avi who should be able to have a look at it today or tomorrow.

**Review by amisstea — APPROVED** [review]
/lgtm

**Pull request merged**